src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -76,14 +76,33 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
             if (File.Exists(documentationPath))
                 return true;
 
+            //try with uppercase XML for linux
+            documentationPath = PathHelper.replaceExtension(documentationPath, ".XML");
```

Comment:
What I was thinking was find all files named documentationPath regardless of extension, and then check them for case-insensitive extension of XML.  This would find it regardless of case, but would probably be ambiguous which one would be picked if there are multiple.